### PR TITLE
Fixed letter to number conversion in search

### PR DESCRIPTION
**Pull request by Aga-C**
Moved fix from https://github.com/FossifyOrg/Contacts/pull/17

### CODE REVIEW

File: app/src/main/kotlin/org/fossify/dialer/fragments/ContactsFragment.kt [hunk 1]
```diff
@@ -153,9 +153,9 @@ class ContactsFragment(context: Context, attributeSet: AttributeSet) : MyViewPag
         val filtered = allContacts.filter {
             getProperText(it.getNameToDisplay(), shouldNormalize).contains(fixedText, true) ||
                 getProperText(it.nickname, shouldNormalize).contains(fixedText, true) ||
-                it.phoneNumbers.any {
+                (fixedText.toDoubleOrNull() != null && it.phoneNumbers.any {
```
Comment (naveensingh) [inline app/src/main/kotlin/org/fossify/dialer/fragments/ContactsFragment.kt:156]: Is there any reason why it's not `toIntOrNull()`? Phone numbers usually don't contain decimals.

**Comment by naveensingh** [comment]
Search still works in RecentsFragment after converting my phone number to letters.

**Comment by Aga-C** [comment]
I've changed `toDouble` to `toInt` and also fixed search in Recents. Additionally, I've also added previous spaces fix there.

**Comment by Aga-C** [comment]
Yes, it's ready.

**Pull request merged**